### DISCUSSION TITLE
WRONG MOBILE & DESKTOP UNIVERSAL LINK FOR LEAD WALLET APP SUBMISSION #454

### DISCUSSION THREAD

**Discussion by Francisifegwu** [WalletConnect/walletconnect-monorepo#896] [Developer Support]
Hi @chadyj please I accidentally added the wrong universal link to our wallet connect registry application. Kindly advise on how I can fix this, as the issue has already been closed and archived by you. Thanks.

Issue link: https://github.com/WalletConnect/walletconnect-registry/issues/454

The correct Desktop and Mobile Universal Linking: https://ws.leadwallet.io/api/api/v1/wallet-connect/

Thank you for all you do heart

**Comment by chadyj** [comment]
@Francisifegwu You can now manage your own registry listing at https://walletconnect.com

Ping me on Discord and I can migrate your account.

**Reply by Francisifegwu** [reply]
Sure. Are you using the same handle here?

**Reply by Francisifegwu** [reply]
@chadyj We've already set up a walletconnect registry with our Dev email - dev@leadwallet.io. For some reason I can't find you on Discord. Maybe you can just send me an email (dev@leadwallet.io) on the next step. Thank you 😊

**Reply by Francisifegwu** [reply]
Thank you so much for the help
